### DISCUSSION TITLE
Updated Pychron, no GitHost Plug In

### DISCUSSION THREAD

**Discussion by gennachiaro** [NMGRL/pychron#2227] [Q&A]
I just updated my Pychron and am getting the message "No GitHost Plugin enabled". How should I go about enabling this? 

<img width="424" alt="Screenshot 2023-10-06 at 6 54 13 PM" src="https://github.com/NMGRL/pychron/assets/56849646/926225cd-ef4f-4507-87ba-de5d44643faf">

Here is my current log file: [pychron.current.log](https://github.com/NMGRL/pychron/files/12836038/pychron.current.log)

**Comment by jirhiker** [comment]
You can enable the GitHub plugin using https://github.com/NMGRL/pychron/wiki/faq#general

goto `Menu/Help/Edit Initialization`.  Then double click on GitHub to enable

**Reply by gennachiaro** [reply]
It was already enabled when I went to check. I enabled GitLab too and the message went away, but I get this message as well (I think I also got this message before as well, when GitLab wasn't enabled, but GitHub was):
<img width="424" alt="Screenshot 2023-10-10 at 4 52 33 PM" src="https://github.com/NMGRL/pychron/assets/56849646/d08147e5-95bb-462e-ae14-df7fc32aebe7">

**Reply by jirhiker** [reply]
ok disable the GitLab plugin.  You just need the GitHub plugin. 

To fix the issue(s),  install `requests_oauthlib` into your pychron python environment
